### PR TITLE
Document `isSet` and `unset` for scalar lists

### PR DESCRIPTION
**Pull request by keerlu**
Document `isSet` and `unset` for scalar lists

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx [hunk 1]
```diff
@@ -55,9 +55,43 @@ const createdUser = await prisma.user.create({
...
+
+Available for:
+
+- PostgreSQL in versions [2.15.0](https://github.com/prisma/prisma/releases/tag/2.15.0) and later
+- CockroachDB in versions [3.9.0](https://github.com/prisma/prisma/releases/tag/3.9.0) and later
+- MongoDB in versions [3.11.0](https://github.com/prisma/prisma/releases/tag/3.11.0) and later
+
+</Admonition>
+
+Use the atomic [`push`](/reference/api-reference/prisma-client-reference#push) <span class="api"></span> method to add a single value to a scalar list:
```
Comment (dpetrick) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx]: In general I'd be careful to call things atomic, the push itself is atomic, the operation as a whole could be argued to not be atomic - I'd just leave out `atomic`.

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx [hunk 2]
```diff
@@ -101,7 +135,17 @@ if (user) {
...
+
+Available for:
+
+- PostgreSQL in versions [2.15.0](https://github.com/prisma/prisma/releases/tag/2.15.0) and later
+- CockroachDB in versions [3.9.0](https://github.com/prisma/prisma/releases/tag/3.9.0) and later
+- MongoDB in versions [3.11.0](https://github.com/prisma/prisma/releases/tag/3.11.0) and later
+
+</Admonition>
+
+Use [scalar list filters](/reference/api-reference/prisma-client-reference#scalar-list-filters) <span class="api"></span> to search within scalar lists. The following example returns all posts where the tags list includes `databases` _and_ `typescript`:
```
Comment (dpetrick) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx]: Nitpick: "search within scalar lists" is misleading. You're filtering model _records_ based on scalar list filters like "is this list empty",  "does this list have value X"... not sure `search` is the best generalization here, but I may be wrong.

File: content/400-reference/200-api-reference/050-prisma-client-reference.mdx [hunk 3]
```diff
@@ -4107,13 +4107,46 @@ const addTag = await prisma.post.update({
...
+### <inlinecode>unset</inlinecode>
+
+<Admonition type="warning">
+
+This method is available on MongoDB only in versions
+[3.11.1](https://github.com/prisma/prisma/releases/tag/3.11.1) and later.
+
+</Admonition>
+
+Use `unset` to unset the value of a scalar list. Unlike `set: null`, `unset` removes the list entirely.
```
Comment (dpetrick) [inline content/400-reference/200-api-reference/050-prisma-client-reference.mdx:4119]: Is the focus on scalar lists here intended?

Comment (dpetrick) [reply content/400-reference/200-api-reference/050-prisma-client-reference.mdx:4119]: (I think it is, I just have a hard time reviewing snippets while needing to expand the sections to figure out where I am)

Comment (keerlu) [reply content/400-reference/200-api-reference/050-prisma-client-reference.mdx:4119]: Yes, intended, it's part of the 'Scalar list methods' subsection

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx (hunk 1 above)
Comment (keerlu) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx]: Thanks, done. Leftover wording from previous version of docs.

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx (hunk 2 above)
Comment (keerlu) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/200-working-with-scalar-lists-arrays.mdx]: Hopefully updated version is better.

**Pull request merged**